scripts/parse_diff_output_from_compare.py

```diff
@@ -0,0 +1,17 @@
+import sys
```

Comment:
Is this script just for your debugging? Or is it useful for users also? i.e should it be included in the PR?

Reply:
Removed at https://github.com/leoisl/pandora/commit/297232b10c7c8d88f08c2b7d260bc7c8d63e6020